com.microsoft.mrtk.input/Interactors/GazePinch/GazePinchInteractor.cs

```diff
@@ -38,7 +38,22 @@ public class GazePinchInteractor :
         /// <summary>
         /// The worldspace pose of the hand pinching point.
         /// </summary>
-        public Pose PinchPose => handController.PinchSelectPose;
+        public Pose PinchPose => pinchPoseSource.TryGetPose(out Pose pinchPose) ? pinchPose : Pose.identity;
```

Comment:
Should this fall back to `handController.PinchSelectPose`?

Reply:
handController.PinchSelectPose is already calls into the same code paths. The sentiment I got was we wanted to reduce our dependency on `ArticulatedHandContoller` specific fields, so I wouldn't want to leave the PinchSelectPose as a fallback.

Reply:
Got it. Seems good then. Should we `[Obsolete]` `handController.PinchSelectPose` to start moving in that direction, or is it still used elsewhere in the repo?

Reply:
I can do so